components/basic-analyzers/src/main/java/org/datacleaner/beans/BooleanAnalyzer.java

```diff
@@ -57,6 +57,7 @@
     public static final String MEASURE_ROW_COUNT = "Row count";
     public static final String DIMENSION_COLUMN = "Column";
     public static final String DIMENSION_MEASURE = "Measure";
+    public static final String DIMENSION_COMBINATION = "Combination ";
```

Comment:
maybe call it DIMENSION_COMBINATION_PREFIX because it is more a prefix than an actual dimension name.